components/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file Skeleton.tsx
+ * Skeleton placeholder component.
```

Comment:
So this is just used while the transcript is loading - and then replaced with the actual transcript element?

Reply:
Yes. It gives the transcript area something to show while it is loaded and rendered, which can take some time due to the size of the files being retrieved and the shear number of elements being rendered.